doc/release-notes/0.37/0.37.md

```diff
@@ -117,20 +117,52 @@ on x64 platforms, but builds with OpenJDK 17 and later also require more stack s
 
 <tr>
 <td valign="top"><a href="https://github.com/eclipse-openj9/openj9/issues/13767">#13767</a></td>
-<td valign="top">Compressed references mode is not available</td>
+<td valign="top">Compressed references mode is not available.</td>
 <td valign="top">Apple silicon macOS</td>
 <td valign="top">You can use only the large heap (non-compressed references) mode.</td>
 <td valign="top">None</td>
 </tr>
 
 <tr>
 <td valign="top"><a href="https://github.com/eclipse-openj9/openj9/issues/14803">#14803</a></td>
-<td valign="top">Using the <tt>-XX:+ShowHiddenFrames</tt> option in an OpenJ9 release that is built with OpenJDK 18 and later causes errors due to underlying problems in OpenJDK</td>
+<td valign="top">Using the <tt>-XX:+ShowHiddenFrames</tt> option in an OpenJ9 release that is built with OpenJDK 18 and later causes errors due to underlying problems in OpenJDK.</td>
 <td valign="top">All platforms</td>
 <td valign="top">Wrong exception might be thrown when using the Reflection API.</td>
 <td valign="top">Avoid using the <tt>-XX:+ShowHiddenFrames</tt> option with OpenJDK 18 and later.</td>
 </tr>
 
+<tr>
+<td valign="top"><a href="https://github.com/eclipse-openj9/openj9/issues/16728">#16728</a></td>
+<td valign="top">Virtual threads may cause slowness and errors under heavy workloads due to underlying problems in OpenJ9.</td>
```

Comment:
Just a thought....I'm not sure adding "due to underlying problems in OpenJ9" for every single issue is all that important to call out, is it? Won't people naturally assume these problems in an OpenJ9 release are due to OpenJ9?

Reply:
Put another way: the point of this list is to highlight issues, not to assign blame for those issues